SwrveConversationSDK/Conversation/SwrveMessageEventHandler.h

```diff
@@ -0,0 +1,16 @@
+#import <Foundation/Foundation.h>
+
+@class SwrveBaseConversation;
+
+@protocol SwrveMessageEventHandler <NSObject>
```

Comment:
Thought we said SwrveConversationEventHandler. Message events pertains to In App Message events. These are purely conversation events.